lib/libraries/cms/controller/form.php

```diff
@@ -298,7 +298,7 @@ public function cancel($key = null)
 		$app = JFactory::getApplication();
 		$model = $this->getModel();
 		$table = $model->getTable();
-		$checkin = property_exists($table, 'checked_out');
```

Comment:
@johanjanssens This changes the logic from "has checked_out field in the table?" to "is the current row checked out?" right? 

And since the table object does not have a row loaded, it will always return false, therefore not running the code block that follows. Or am I reading it wrong?